pipeline-runner/R/qc-helpers.R

```diff
@@ -200,7 +200,7 @@ getClusters <- function(clustering_method, resolution, data) {
 
   if (clustering_method == "leiden") {
     # emulate FindClusters, which overwrites seurat_clusters slot and meta.data column
-    g <- getSNNiGraph(data)
+    g <- getSNNiGraph(data, active.reduction)
```

Comment:
Can you rename this "g" to something more meaningful? 
I see that this `getSNNiGraph` function was used before too, but it wasn't defined before. Was there a built-in function in Seurat, called `getSNNiGraph` that we were using in the past in this case?

Reply:
I copied it from the worker verbatim. This function missing seems to be the reason Leiden clustering was not working in the pipeline (now the clustering tests with leiden pass). And sure, I'll rename to snn_graph.

Reply:
sure! I'll rename in the worker as well